app/src/main/java/com/nervousfish/nervousfish/service_locator/EntryActivity.java

```diff
@@ -21,6 +26,7 @@
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        LOGGER.info("EntryActivity entered");
```

Comment:
Maybe change this to `EntryActivity created` and move it to the end of the method for clarity